examples/nft-dumb/index.rsh

```diff
@@ -26,28 +26,20 @@ export const main =
     while ( true ) {
       commit();
 
-      Owner.only(() => {
```

Comment:
This doesn't quite capture what the original contract does. If we follow the old logic:

```
NFT Creator deploys contract
NFT Creator passes id of NFT into contract

[ =    set variable to track new owner    = ]
[ =    start while loop                               = ]

NFT Creator joins contract as Owner ParticipantClass

Owner (NFT Creator) publishes new owner
  FIRST LOOP:
     NFT Creator will specify the address for the new Owner
  ALL OTHER TIMES:
     Owner will be the ParticipantClass member that won the race to join

Contract sets new Owner of NFT to variable

[ =    end while loop                               = ]
```

We can jam on this after all the meetingz™ if you like.